chatting/team/views.py

```diff
@@ -18,7 +17,7 @@ def index(request):
 def channel_create(request):
```

Comment:
I think you need to add `@login_required`
https://docs.djangoproject.com/en/1.8/topics/auth/default/#django.contrib.auth.decorators.login_required